File: JavaScript/exercicio-06/solucao.js
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function main() {

  function antecessor_sucessor() {
    const numero = parseInt(prompt('Digite um número: ' ));
  
    const antecessor = numero - 1;
    const sucessor = numero + 1;
  
    console.log(`O antecessor de ${numero} é ${antecessor}`);
    console.log(`O sucessor de ${numero} é ${sucessor}`); 
  }
  
  antecessor_sucessor();

}

main();
readline.question("Digite Enter para sair ... ");

File: C/exercicio-06/solucao.c
#include <stdio.h>

void antecessorSucessor() {
  int num, antecessor, sucessor;

  printf("Digite um número: " );
  scanf("%d", &num);

  antecessor = num - 1;
  sucessor = num + 1;

  printf("O antecessor de %d é igual a %d \n", num, antecessor);
  printf("O sucessor de %d é igual a %d \n", num, sucessor);

}

int main() {
  antecessorSucessor();
  return 0;
}